src/http/server.ts

```diff
@@ -20,6 +20,18 @@ interface IRequestOptions {
   };
 }
 
+/**
+ * Used to check if a response object is of type Drash.Interfaces.ResponseOutput
+ */
+// @ts-ignore Only exception because response cannot be properly typed and we're checking if it is of type interface anyway
+function responseIsOfTypeResponseOutput(response: any): boolean {
+  if (typeof response === "object" && Array.isArray(response) === false) {
```

Comment:
would it every be an array? i ask because our docs explicitly state to `return this.response`. although this does cover if someone wants to try returning an array.

Reply:
This is more a specific edge case, because `typeof [] === "object"` is `true`,  we're making sure that it is a proper object so there are no uncaught errors

Reply:
for example, `response` could be `"hello"`, and i thought it was best to add the check inside the function so we can do `responseIsOfTypeObject(...)` wherever we want without adding code each time to check if its a proper object

Reply:
cool!